aptos-move/framework/aptos-framework/sources/storage_gas.move

```diff
@@ -0,0 +1,254 @@
+module aptos_framework::storage_gas {
+
+    use aptos_framework::system_addresses;
+    use std::error;
+    use aptos_framework::state_storage;
+    use std::vector;
+
+    friend aptos_framework::gas_schedule;
+    friend aptos_framework::genesis;
+    friend aptos_framework::reconfiguration;
+
+    const ESTORAGE_GAS_CONFIG: u64 = 0;
+    const ESTORAGE_GAS: u64 = 1;
+    const EINVALID_GAS_RANGE: u64 = 2;
+    const EZERO_TARGET_UTILIZATION: u64 = 3;
+    const EINVALID_CURVE_LENGTH: u64 = 4;
+    const EINVALID_CURVE_START_POINT: u64 = 5;
+    const EINVALID_CURVE_END_POINT: u64 = 6;
+    const EINVALID_MONOTONICALLY_NON_DECREASING_CURVE: u64 = 7;
+
+    const BASIS_POINT_BASE: u64 = 10000;
+
+    /// This updates at reconfig and guarantees not to change elsewhere, safe
+    /// for gas calculation.
+    ///
+    /// Specifically, it is updated based on the usage at the begining of the
+    /// current epoch when end it by executing a reconfig transaction. The gas
```

Comment:
end -> ending?
probably easier for the reader if you say "it is .. when .. based on ..